ERCS/erc-7281.md

```diff
@@ -0,0 +1,356 @@
+
+1. Deploy a lockbox on the remote domain which locks the canonical-bridge-minted token and mints a new `XERC20`.
+2. Establish the new `XERC20` token as the canonical asset for the domain - this is largely a social consensus activity driven by DAO agreement and project communications.
+3. Deploy a lockbox on the home domain which locks the home canonical token and mints a new `XERC20`.
+4. Allowlist all desired bridges *including* the canonical bridge that owns the legacy implementation, mapping the `XERC20` representation on the home domain to the `XERC20` representation on the remote.
+5. At this point, it is fully possible for users to begin transferring the token across all bridges as would be expected from the `XERC20` standard. However, note that the canonical bridge connecting home to remote will now have two bridge paths: (a) `ERC20`→`ERC20` (locking the `ERC20` at home), and (b) `XERC20`→`XERC20`. Now, the issuer can begin the process of sunsetting the legacy canonical `ERC20` on the remote domain.
+6. The issuer should at this point disallow minting *new* legacy canonical `ERC20`s on the remote domain. This would organically & gradually lead to the legacy `ERC20`s on the remote chain to become locked into the lockbox.
+7. At any point in the future, the token issuer can then use their own treasury to unwrap `XERC20`s on the remote domain into legacy `ERC20`s and send them back to the home chain, incurring the latency of this on behalf of users.
+
+### Compatibility with Bridges
+
+One key benefit to the `XERC20` model is that the simple requirement of a burn & mint interface makes this proposal compatible with most bridges either out of the box or through a predefined custom token mapping process. The following bridges & domains were considered in the creation of this proposal:
+
+- **3rd party bridges:** every 3rd party bridge that relies on messaging infrastructure already supports a mint and burn interface and has a pathway to map custom minted tokens.
+- **Arbitrum rollup bridge:**  Arbitrum allows token issuers to write a custom gateway
+- **OP Stack bridge:** Optimism supports adding a custom bridge
+- **Polygon PoS bridge:** Polygon supports custom tokens through their fx portal
+- **ZkSync bridge**: Zksync supports custom bridges of any kind
+- **GnosisChain Omnibridge:** GnosisChain does not currently support custom tokens, but is in the process of redesigning their omnibridge and plans to allow this functionality.
+
+Note: In most of the above cases, a custom bridge integration also means integration into canonical bridge UIs, ensuring that users have a consistent experience throughout the process. Additionally, a single `XERC20` custom bridge implementation could be built for each ecosystem and serve any number of token issuers.
+
+## Security Considerations
+
+Please see the associated discussion in the **Rationale** section.
```

Comment:
A reader coming to this document to specifically find security considerations won't really benefit from this sentence. Security considerations should be in this section.